### PR TITLE
Load library current locations and Authors

### PR DESCRIPTION
**Pull request by kehh**
What source and destination author values should look like:

| Original value | Replacement Value |
| --- | --- |
| [Warung], [Price] | Warung, Price |
| _Forster_, Shirley | Forster, Shirley |
| ` |  |
| 4th Aust. Field Survey Corps, R.A.E. | Australian Survey Corps, Field Survey Co. |
| 4th Aust. Field Survey Coy. | Australian Survey Corps. Field Survey Co. |

What source and destination StoredLocation should look like:

| Original | Becomes |
| --- | --- |
| ;IBRARY | Library |
| .LIBRARY ANNEXE | FoyerAnnex |
| `RARE BOOK SHELVES | PhotographsRoom |
| A.E.Williams shelf in foyer | FoyerAnnex |

### CODE REVIEW

File: importScripts/sampleRecords/library.destination.json [hunk 1]
```diff
@@ -264,6 +264,47 @@
...
       "id":"8",
+      "idno":"D2009\/8",
+      "display_label":"Wilfred Priestner",
+      "ca_objects.LegacyID":"10296",
+      "ca_objects.Series":"1",
+      "ca_objects.Size":"Large",
+      "ca_objects.Author":[
+        "Newman, Philippa"
+      ],
+      "ca_objects.StoredLocation":"Library",
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json]: Was `A.E.Williams shelf in foyer`

File: importScripts/sampleRecords/library.destination.json [hunk 2]
```diff
@@ -344,8 +385,47 @@
...
+      "object_id":"11",
+      "id":"11",
+      "display_label":"The golden age of cinema in Perth. (in Early Days, Vol. 8, pt.6, pp. 39 - 53)",
+      "ca_objects.LegacyID":"10560",
+      "ca_objects.Series":"1",
+      "ca_objects.Size":"Octavo",
+      "ca_objects.Author":[
+        "Honniball, J. H. M."
+      ],
+      "ca_objects.StoredLocation":"Library",
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json:397]: Was `.LIBRARY ANNEXE`

File: importScripts/sampleRecords/library.destination.json [hunk 3]
```diff
@@ -382,15 +462,15 @@
...
+      "object_id":"13",
+      "id":"13",
       "display_label":"National estate values in the Southern Forest region of South-West Western Australia",
       "ca_objects.LegacyID":"120",
       "ca_objects.Series":"0",
       "ca_objects.Size":"Large",
       "ca_objects.Author":[
         "Australian Heritage Commission",
-        "Dept of CALM"
+        "Western Australia. Dept. of Conservation and Land Management"
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json:473]: This one was also fixed!

File: importScripts/sampleRecords/library.destination.json [hunk 4]
```diff
@@ -490,15 +570,15 @@
...
+      "object_id":"16",
+      "id":"16",
       "idno":"2011\/186",
       "display_label":"A History of the Independent Schools Salaried Officers' Association  of W.A. Industrial Union of Workers, 1960 - 1976",
       "ca_objects.LegacyID":"13359",
       "ca_objects.Series":"0",
       "ca_objects.Size":"Large",
       "ca_objects.Author":[
-        "Hiller, J  M"
+        "Hiller, J. M."
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json:581]: ditto

File: importScripts/sampleRecords/library.destination.json [hunk 5]
```diff
@@ -529,8 +609,46 @@
...
     {
-      "object_id":"15",
-      "id":"15",
+      "object_id":"17",
+      "id":"17",
+      "display_label":"Peelers Gazette; newsletter (of)  The Western Australia Police Historical Society Inc.",
+      "ca_objects.LegacyID":"13408",
+      "ca_objects.Series":"0",
+      "ca_objects.Author":[
+
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json:618]: was "`"

File: importScripts/sampleRecords/library.destination.json [hunk 6]
```diff
@@ -264,6 +264,47 @@
...
+      "ca_objects.PublicationType":"Journal title",
+      "ca_objects.DateOfPublication":"2008",
+      "ca_objects.Publisher":"Author",
+      "ca_objects.PlaceOfPublication":"Perth",
+      "ca_occurrences":[
+        "Art metal-work",
+        "Priestner, Wilfred",
+        "Wrought iron"
+      ],
+      "ca_objects.CurrentLocation":"FoyerAnnex",
```
Comment (kehh) [inline importScripts/sampleRecords/library.destination.json:289]: Was `A.E.Williams shelf in foyer`

**Comment by mostro20** [comment]
I can't believe how much work has gone into your scripts. Impressive.

**Pull request merged**